README.md

```diff
@@ -5,25 +5,30 @@
 <br><br>
 <a href="https://npmjs.com/package/spectacle"><img src="https://img.shields.io/npm/dm/spectacle.svg"></a>
 <a href="https://npmjs.com/package/spectacle"><img src="https://img.shields.io/npm/v/spectacle.svg"></a>
-<img src="http://img.badgesize.io/https://unpkg.com/spectacle/dist/spectacle.min.js?compression=gzip&label=gzip%20size">
 <a href="https://github.com/FormidableLabs/spectacle#maintenance-status">
   <img alt="Maintenance Status" src="https://img.shields.io/badge/maintenance-active-green.svg" />
 </a>
 </p>
 
-Looking for a quick preview of what you can do with Spectacle? Check out our Live Demo deck
-[here](https://raw.githack.com/FormidableLabs/spectacle/main/examples/one-page.html).
+## Getting Started
+
+Welcome to our monorepo project, housing the core [`spectacle` package](https://github.com/FormidableLabs/spectacle/blob/main/packages/spectacle/README.md) and related tools and examples.
+
+Come learn more at https://www.formidable.com/open-source/spectacle!
```

Comment:
Should we make this an explicit anchor tag with `[...](...)`?

Reply:
Yep!